meson.build

```diff
@@ -572,6 +579,7 @@ assert(long_max > 100000)
 conf.set_quoted('LONG_MAX_STR', f'@long_max@')
 
 foreach ident : [
+        ['renameat2',         '''#include <stdio.h>'''],        # since musl-1.2.6
```

Comment:
uh, I'd prefer to avoid adding even more version checks. I mean, this is already merged, and it's the second class libc that won't get full support in v259. Can't we just wait for the new release?

Reply:
Unfortunately musl does not release new version so often. In the series of the commits, I only requires the latest musl release, and v1.2.5 is the latest, unfortunately.

Reply:
Note, this has the same review history as the other place. I initially implemented in a generic way like this, and Luca did not like that, then I conditionalized this is checked only when musl is used, but Lennart requested to make this generic, I dropped the conditonalization, then now you do not like it...

I will merge this as is at least now. We can come back later if this really causes any measurable performance regression or so.